documentation/src/docs/getting-started.md

```diff
@@ -29,7 +29,7 @@ If you prefer to build Pravega yourself, you can download the code and run `./gr
 details are shown in the Pravega [README.md](https://github.com/pravega/pravega/blob/master/README.md).
 
 ```
-$ tar xfvz pravega-0.1.0.tgz
+$ tar xfvz pravega-<version>.tgz
```

Comment:
Just so that I understand, are we populating this version or are we leaving it generic?

Reply:
we are leaving it generic.